### PR TITLE
Bug 1855575: Bump golang to 1.14 for kas-o, kcm-o, kd-o, cpc

### PR DESCRIPTION
**Pull request by soltysh**
/assign @damemi 

The repo PRs are: 
- https://github.com/openshift/cluster-kube-apiserver-operator/pull/929
- https://github.com/openshift/cluster-kube-descheduler-operator/pull/118
- https://github.com/openshift/cluster-policy-controller/pull/35

### CODE REVIEW

**Comment by soltysh** [comment]
/hold
this is being discussed within group b now.

**Comment by soltysh** [comment]
Ok, go1.14 if for now, let's move with this.

File: ci-operator/config/openshift/cluster-kube-descheduler-operator/openshift-cluster-kube-descheduler-operator-master.yaml [hunk 1]
```diff
@@ -11,13 +15,13 @@ build_root:
...
-    tag: golang-1.13
+    tag: golang-1.14
 images:
 - from: base
   inputs:
-    openshift_release_golang-1.13:
+    openshift_release_golang-1.14:
       as:
-      - registry.svc.ci.openshift.org/openshift/release:golang-1.13
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.14
```
Comment (oshoval) [inline ci-operator/config/openshift/cluster-kube-descheduler-operator/openshift-cluster-kube-descheduler-operator-master.yaml:24]: Hi,
why some have this URL and most have 
registry.svc.ci.openshift.org/ocp/builder ?

i saw this discussion once
https://github.com/openshift/release/pull/10224#discussion_r454412103

**Pull request merged**